### PR TITLE
Merge changes from dev workshop review branch to master

### PR DESCRIPTION
**Pull request by Anvesha4**
Merging dev review workshops with master. Will address visibility and draft values of the workshops in a different PR

### CODE REVIEW

File: content/english/adafruit/_index.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ prereq: "None"
 difficulty: "Beginner"
 download: ""
 draft: false
+hidden: true
```
Comment (ozhang22) [inline content/english/adafruit/_index.md:9]:
```suggestion
```

File: content/english/adventure/_index.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ difficulty: Intermediate
 prereq: "Python Basics - Writing to Console (Print), Reading from Console, Strings, Variables, Conditional statements"
 download: ""
 draft: false
+hidden: true
```
Comment (ozhang22) [inline content/english/adventure/_index.md:9]:
```suggestion
```

File: content/english/arduino/Circuit-Basics.md [hunk 3]
```diff
@@ -1,11 +1,12 @@
 ---
-title: "Circuit Basics"
+title: "Appendix"
```
Comment (ozhang22) [inline content/english/arduino/Circuit-Basics.md:2]: Can you rename the file too?

File: content/english/arduino/Circuit-Basics.md [hunk 4]
```diff
@@ -17,16 +18,17 @@ Imagine our circuit is a play. Before the final performance in front of an audie
 + The same is true on the red and blue lines on the left. However, the red and blue lines on the right and left are not connected to each other.
 + Components that are plugged into the same common connection are connected in the circuit
 
-![Alt Text: Picture of a breadboad](../img/breadboard.png)
+![Cannot load image](../img/img16.png)
```
Comment (ozhang22) [inline content/english/arduino/Circuit-Basics.md:21]: I think you want to undo the changes on this page.

File: content/english/arduino/_index.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ date: 2020-07-27T14:04:59.61-07:00
 difficulty: "Intermediate"
 icon: "fas fa-microchip"
 draft: false
-hidden: false
+hidden: true
```
Comment (ozhang22) [inline content/english/arduino/_index.md:8]:
```suggestion
```

File: content/english/csharp-guess-the-word/_index.md [hunk 6]
```diff
@@ -5,6 +5,7 @@ date: 2019-07-23T11:45:38-07:00
 prereq: "None"
 difficulty: "Intermediate"
 draft: false
+hidden: true
```
Comment (ozhang22) [inline content/english/csharp-guess-the-word/_index.md:8]:
```suggestion
```

File: content/english/java-basics/_index.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ prereq: "none"
 difficulty: "Beginner"
 icon: "fab fa-java"
 draft: false
-hidden: false
+hidden: true
```
Comment (ozhang22) [inline content/english/java-basics/_index.md:9]:
```suggestion
```

File: content/english/java-tictactoe/_index.md [hunk 8]
```diff
@@ -6,7 +6,7 @@ prereq: "Java: Basics"
 difficulty: "Intermediate"
 icon: "fab fa-java"
 draft: false
-hidden: false
+hidden: true
```
Comment (ozhang22) [inline content/english/java-tictactoe/_index.md:9]:
```suggestion
```

File: content/english/javascript-basics/introduction.md [hunk 9]
```diff
@@ -2,6 +2,7 @@
 title: "Introduction"
 date: 2020-07-28T11:45:38-07:00
 draft: false
+hidden: true
```
Comment (ozhang22) [inline content/english/javascript-basics/introduction.md:5]:
```suggestion
```

File: content/english/microbit-distance-checker/_index.md [hunk 10]
```diff
@@ -6,6 +6,7 @@ prereq: "None"
 difficulty: "Beginner"
 download: ""
 draft: false
+hidden: true
```
Comment (ozhang22) [inline content/english/microbit-distance-checker/_index.md:9]:
```suggestion
```

File: content/english/microbit/_index.md [hunk 11]
```diff
@@ -5,6 +5,7 @@ date: 2019-07-23T14:54:53-07:00
 difficulty: "Beginner to Intermediate"
 download: "https://github.com/NuevoFoundation/workshops/tree/master/content/microbit"
 draft: true
+hidden: true
```
Comment (ozhang22) [inline content/english/microbit/_index.md:8]:
```suggestion
```

File: content/english/python-earsketch/_index.md [hunk 12]
```diff
@@ -6,6 +6,7 @@ prereq: "Python Basics: Print, Comments, Functions"
 difficulty: "Intermediate"
 download: ""
 draft: false
+hidden: true
```
Comment (ozhang22) [inline content/english/python-earsketch/_index.md:9]:
```suggestion
```

File: content/english/python-guess-number/_index.md [hunk 13]
```diff
@@ -6,7 +6,7 @@ prereq: "Python Basics"
 difficulty: "Intermediate"
 icon: "fab fa-python"
 draft: false
-hidden: false
+hidden: true
```
Comment (ozhang22) [inline content/english/python-guess-number/_index.md:9]:
```suggestion
```

File: content/english/python-pixel/_index.md [hunk 14]
```diff
@@ -5,6 +5,7 @@ prereq: "Python Basics"
 difficulty: "Intermediate"
 icon: "fab fa-python"
 draft: false
+hidden: true
```
Comment (ozhang22) [inline content/english/python-pixel/_index.md:8]:
```suggestion
```

File: content/english/python-tensorflow/_index.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Python: Machine Learning Basics using Tensorflow"
-description: "Learn basic image classification and train ML models with Tensorflow"
+title: "Python: Machine Learning Basics using TensorFlow"
```
Comment (ozhang22) [inline content/english/python-tensorflow/_index.md:2]: Make sure these changes don't conflict with @beagandica

File: content/english/python-tensorflow/building_neural_network.md [hunk 16]
```diff
@@ -28,53 +24,32 @@ Take a look at these two images.
 
 As we can see from these images, we have two sneakers. How do we know this, even though the images themselves and their pixel values vary significantly? Our brain interprets these images by looking at patterns we observed before and says "That is a sneaker!"
 
-And why is this called a sneaker? Everyone at some point has collectively agreed that this is different from other types of shoes because sneakers have laces, long snouts, thick rubber soles, and cover your feet upto your ankles. This is what we were probably told as kids- to identify these types of shoes as sneakers.
+And why is this called a sneaker? Everyone at some point has collectively agreed that this is different from other types of shoes because sneakers have laces, long snouts, thick rubber soles, and cover your feet upto your ankles. This is what we were probably told as kids - to identify these types of shoes as sneakers.
 
 How is this relevant to what we are doing next? We will need to tell the computer that these types of images are sneakers. We cannot just hard code these two images as sneakers because this would be ineffective when predicting future inputs using this model.
```
Comment (ozhang22) [inline content/english/python-tensorflow/building_neural_network.md:29]: We don't really define what "hard code" means here...

File: content/english/python-turtle/_index.md [hunk 17]
```diff
@@ -6,6 +6,7 @@ prereq: "Python Basics"
 difficulty: "Beginner"
 image: ""
 draft: false
+hidden: true
```
Comment (ozhang22) [inline content/english/python-turtle/_index.md:9]:
```suggestion
```

File: content/english/security/_index.md [hunk 18]
```diff
@@ -5,6 +5,7 @@ date: 2020-02-217T11:45:38-07:00
 prereq: "None"
 difficulty: "Beginner"
 draft: false
+hidden: true
```
Comment (ozhang22) [inline content/english/security/_index.md:8]:
```suggestion
```

File: content/english/web-basics/_index.md [hunk 19]
```diff
@@ -3,6 +3,7 @@ title: "HTML & CSS: Web Basics"
 description: "An introductory guide on how to make a simple website from scratch"
 date: 2019-09-03T17:53:41-07:00
 draft: false
+hidden: true
```
Comment (ozhang22) [inline content/english/web-basics/_index.md:6]:
```suggestion
```

**Review by ozhang22 — COMMENTED** [review]
Most changes are just to remove the hidden changes

**Review by ozhang22 — APPROVED** [review]

File: content/english/arduino/Circuit-Basics.md (hunk 3 above)
Comment (Anvesha4) [reply content/english/arduino/Circuit-Basics.md:2]: done

File: content/english/arduino/Circuit-Basics.md (hunk 4 above)
Comment (Anvesha4) [reply content/english/arduino/Circuit-Basics.md]: removed img17,18,19, and added img16

File: content/english/python-tensorflow/_index.md (hunk 15 above)
Comment (Anvesha4) [reply content/english/python-tensorflow/_index.md:2]: don't see any merge conflicts for this file, resolving

File: content/english/python-tensorflow/building_neural_network.md (hunk 16 above)
Comment (Anvesha4) [reply content/english/python-tensorflow/building_neural_network.md]: replaced "we cannot just hard code" with plain english like "we cannot just mark"
